drone/client.go

```diff
@@ -229,6 +229,15 @@ func (c *client) Build(owner, name string, num int) (*Build, error) {
 	return out, err
 }
 
+// Manually build.
```

Comment:
From https://golang.org/doc/effective_go.html#commentary :

> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

You should start your comment with "BuildCreate"

Reply:
thanks